lib/getIpfsLink.ts

```diff
@@ -0,0 +1,6 @@
+const getIpfsLink = (hash: string) =>
+  hash?.indexOf?.("ipfs://") > -1
+    ? hash.replace("ipfs://", "https://cloudflare-ipfs.com/ipfs/")
```

Comment:
you need to use `thirdweb` gateway right?
- or, were you able to get `cloudflare` gateway to work?